docs/tutorials/9_compare_baselines.ipynb

```diff
@@ -0,0 +1,429 @@
+    "\n",
+    "expert_bc_trainer = bc.BC(\n",
+    "    observation_space=env.observation_space,\n",
+    "    action_space=env.action_space,\n",
+    "    demonstrations=expert_transitions,\n",
+    "    rng=rng,\n",
+    ")\n",
+    "\n",
+    "not_expert_bc_trainer = bc.BC(\n",
+    "    observation_space=env.observation_space,\n",
+    "    action_space=env.action_space,\n",
+    "    demonstrations=not_expert_transitions,\n",
+    "    rng=rng,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "expert_bc_trainer.train(n_epochs=1)\n",
+    "not_expert_bc_trainer.train(n_epochs=1)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bc_expert_rewards, _ = evaluate_policy(\n",
+    "    expert_bc_trainer.policy, env, 10, return_episode_rewards=True\n",
+    ")\n",
+    "bc_not_expert_rewards, _ = evaluate_policy(\n",
+    "    not_expert_bc_trainer.policy, env, 10, return_episode_rewards=True\n",
+    ")\n",
+    "significant = is_significant_reward_improvement(\n",
+    "    bc_not_expert_rewards, bc_expert_rewards, 0.05\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(f\"Cloned expert rewards: {bc_expert_rewards}\")\n",
+    "print(f\"Cloned not-expert rewards: {bc_not_expert_rewards}\")\n",
+    "\n",
+    "print(\n",
+    "    f\"Cloned expert is {'NOT ' if not significant else ''}significantly better than the cloned not-expert.\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can also use the same method to compare different algorithms. While CartPole is pretty easy, we can make it more difficult by decreasing the number of episodes in our dataset, and generating them with a suboptimal policy:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "rollouts = rollout.rollout(\n",
+    "    not_expert,\n",
+    "    DummyVecEnv([lambda: RolloutInfoWrapper(env)]),\n",
+    "    rollout.make_sample_until(min_timesteps=None, min_episodes=1),\n",
+    "    rng=rng,\n",
+    ")\n",
+    "transitions = rollout.flatten_trajectories(rollouts)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's try training a behavior cloning algorithm on this dataset.\n",
+    "\n",
+    "Note that for DAgger, we have to cheat a little bit -- it's allowed to use the expert policy to generate additional data.\n",
+    "For the purposes of this tutorial, we'll stick with this to avoid spending hours training an expert for a more complex environment.\n",
+    "\n",
+    "So while this little experiment isn't definitive proof that DAgger is better than BC, you can use the same method to compare any two algorithms."
```

Comment:
In practice we'd also want to at least run the algorithms with several random seeds. And probably do significance tests on the mean returns across seeds rather than mean returns across rollouts.

I'm not sure the tutorial should demonstrate that exactly -- it might be too slow, although we could always just write the code and only do ~n=3 seeds and have a comment saying more are needed to reach significance. But I'd want to at least flag that this this test alone isn't enough to actually conclude one algo is better than another.

Reply:
Also addressed in the commentary, at least for now